WordGame/Extension/StoryboardHelper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  StoryboardHelper.swift
+//  WordGame
+//
+//  Created by Kumari Ritu Pal on 02/08/22.
+//
+
+import UIKit
+// This enum is to get the storyboard
+enum StoryboardName: String {
+    case main = "Main"
+    
+    var storyboard: UIStoryboard {
+        return UIStoryboard(name: self.rawValue, bundle: nil)
+    }
+    
+    func instantiateInitialViewController<T: UIViewController>() -> T {
+        return storyboard.instantiateInitialViewController() as! T
+    }
+}
+
+// This protocol is to instantiate the view controller of the storybard
+protocol StoryboardHelper {
+    static func instantiate<T: UIViewController>(_ type: T.Type, storyboard storyboardName: StoryboardName, identifier: String?, isPad: Bool?) -> T
+}
+
+extension StoryboardHelper where Self: UIViewController {
```

Comment:
another way to do this is 
``` swift
extension UIViewController: StoryboardHelper { }
```
simple to read as extension on `UIViewController` class
Also you can rename this `StoryboardHelper` file as `UIViewController+Instantiable` by looking at your other extension naming.